nixops/ssh_util.py

```diff
@@ -169,6 +183,9 @@ def register_host_fun(self, host_fun: Callable[[], str]) -> None:
         self._host_fun = host_fun
 
     def _get_target(self, user: Optional[str] = None) -> str:
+        if user is None:
```

Comment:
Note: This assertion is a temporary addition. Ideally `user` should not be `Optional`.